lib/ansible/module_utils/facts/utils.py

```diff
@@ -36,9 +36,9 @@ def get_file_content(path, default=None, strip=True):
     return data
 
 
-def get_file_lines(path):
+def get_file_lines(path, strip=True):
```

Comment:
Wouldn't hurt to add unit tests for util.get_file_lines() now.

There are several unit tests that mock out get_file_content so it is easy to mock. Although testing get_file_lines() with a mocked get_file_content doesnt test much of get_file_lines, easy enough test to add.

(get_file_content could use tests as well, though get_file_content is a little more finicky. get_file_content tests do not need  to be part of this pr though...)

Reply:
Can I do this in a separate PR? I'd like to get this one merged since it's a pretty important fix.